demo/transcribe_app.py

```diff
@@ -112,7 +112,7 @@ def setup_gradio_demo():
             with gr.Column():
                 local_model = gr.Textbox(
                     label="Option 3: Paste local path to model directory",
-                    placeholder="artifacts/my-whisper-tiny",
+                    placeholder="artifacts/my-whisper-tiny  NOTE: Not available on HF Spaces",
```

Comment:
minor: You could actually set an env var in huggingface https://huggingface.co/spaces/mozilla-ai/transcribe/settings named like `HF_SPACE` or something, set it to `TRUE` and then check in the code for that environment variable.
So inside HF you wouldn't render the column at all

Reply:
oh good tip! I ll do that actually

Reply:
@daavoo seems to [work](https://huggingface.co/spaces/mozilla-ai/transcribe)!